experiment/methods/src/pysr_install.sh

```diff
@@ -0,0 +1,5 @@
+#install pysr
+
+
+pip install git+git://github.com/MilesCranmer/PySR.git@v0.7.0a2
```

Comment:
rather than pip install here, you can move this to environment.yml following the syntax above.